Kernel/init.cpp

```diff
@@ -144,6 +144,9 @@ extern "C" UNMAP_AFTER_INIT [[noreturn]] void init()
     InterruptManagement::initialize();
     ACPI::initialize();
 
+    // Initialize the PCI Bus as early as possible, for early boot (PCI based) serial logging
+    PCI::initialize();
```

Comment:
Wait, but now we have two calls to `PCI::initialize()`?

Reply:
Whoops, looks like rebasing botched that up, will fix